src/main/resources/templates/index.html

```diff
@@ -45,7 +47,13 @@ <h1>Lukuvinkki</h1>
                     <td th:text="${tip.author}">author</td>
                     <td th:text="${tip.description}">description</td>
                     <td th:text="${tip.checked}">checked</td>
+                    <td th:text="${tip.checkedtime}">checkedtime</td>
                     <!-- <td><a>Tiedot</a></td> Tästä mennään lukuvinkin yksityissivulle, jossa on sen kaikki tiedot -->
+                    <td th:id="'luettu' + ${tip.id}">
+                        <a th:href="@{~/allTips/markasread/{id}(id=${tip.id})}" class="btn btn-success" role="button">
+                            Merkitse lukuvinkki luetuksi
```

Comment:
Olisikohan järkevämpää, jos tässä olisi "Merkitse lukuvinkki lukemattomaksi" kun lukuvinkki on jo merkattu luetuksi? Tai yksinkertaisempi olisi "Merkitse lukuvinkki luetuksi/lukemattomaksi".
```suggestion
                            Merkitse lukuvinkki luetuksi/lukemattomaksi
```